packages/twenty-front/src/modules/object-record/graphql/types/RecordGqlOperationFilter.ts

```diff
@@ -134,8 +134,13 @@ export type LeafObjectRecordFilter = {
   [fieldName: string]: LeafFilter;
 };
 
+export type SearchObjectRecordFilter = {
```

Comment:
As mentioned in the Backend code review, I don't think we need this in the Filter type? At least not yet 🤔 FindMany won't interpret it.

Reply:
yes thanks that was a relic from previous implementation ideas